package.json

```diff
@@ -80,5 +82,11 @@
     "hooks": {
       "pre-push": ".githooks/pre-push & npm run lint & npm run test:ci"
     }
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "src/**/*.{ts,tsx}",
+      "!src/**/*.stories.tsx"
+    ]
   }
-}
+}
```

Comment:
A newline is missing here.
I think if you turn on `"editor.formatOnSave": true,` in your VScode settings, it will automatically handle it for you. Let me know if this doesn't work